digits/model/tasks/torch_train.py

```diff
@@ -513,27 +514,22 @@ def classify_one(self, image, snapshot_epoch=None, layers=None):
             args.append('--labels=%s' % self.dataset.path(self.dataset.labels_file))
             args.append('--mean=%s' % self.dataset.path(constants.MEAN_FILE_IMAGE))
             args.append('--allPredictions=no')
-            if self.use_mean:
-                args.append('--subtractMean=yes')
-            else:
-                args.append('--subtractMean=no')
         elif isinstance(self.dataset, dataset.GenericImageDatasetJob):
-            if self.use_mean:
+            if self.use_mean != 'none':
                 args.append('--mean=%s' % os.path.join(self.job_dir, constants.MEAN_FILE_IMAGE))
-                args.append('--subtractMean=yes')
-            else:
-                args.append('--subtractMean=no')
             args.append('--allPredictions=yes')
 
         if snapshot_epoch:
             args.append('--epoch=%d' % int(snapshot_epoch))
-        if TORCH_USE_MEAN_PIXEL:
-            args.append('--useMeanPixel=yes')
         if self.trained_on_cpu:
             args.append('--type=float')
 
-        # input image has been resized to network input dimensions by caller
-        args.append('--crop=no')
```

Comment:
that `--crop=no` has disappeared but it isn't an issue because that flag defaults to `no`